GenespotRE/settings.py

```diff
@@ -294,17 +282,34 @@ def GET_BQ_COHORT_SETTINGS():
     'allauth.socialaccount',
     'allauth.socialaccount.providers.google')
 
-TEMPLATE_CONTEXT_PROCESSORS = (
-    'allauth.socialaccount.context_processors.socialaccount',
-    'django.core.context_processors.request',
-    'django.contrib.auth.context_processors.auth',
-    'django.contrib.messages.context_processors.messages',
-    'django.core.context_processors.tz'
-)
-
-TEMPLATE_DIRS += (
-    os.path.join(BASE_DIR, 'templates', 'accounts'),
-    )
+# Template Engine Settings
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'OPTIONS': {
+            # add any context processors here
+            'context_processors': (
+                'allauth.socialaccount.context_processors.socialaccount',
```

Comment:
I'm not sure. It may have been something we *tried* and only half removed? If we're not using the context_processor, does that mean there are other things that we can remove? thinking about it now, I suspect the only time we use the django account login is for the admin area. I'd say let's leave it for now, and if you want, you can investigate this further.